seqr/views/utils/orm_to_json_utils.py

```diff
@@ -764,7 +764,7 @@ def get_json_for_rna_seq_outliers(models, max_significant_num_per_tissue=None, *
     if max_significant_num_per_tissue:
         models = models.annotate(place=Window(
             expression=DenseRank(),
-            partition_by=[F("sample__tissue_type")],
+            partition_by=[F('sample__tissue_type'), F('sample__individual_id')],
```

Comment:
It is good for the previous PR because there is always a single individual in it. This PR has to fix it because the variant API needs to return the RnaSeq data on a family basis.